exporter-scripts/entrypoint.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Set up cron schedule according to the environment variable
+if [ -z "$EXPORTER_CRON_SCHEDULE" ]; then
+  echo "You must set EXPORTER_CRON_SCHEDULE when running this container"
+  exit 1
+fi
+set -x
+cat <<EOF | crontab -
+PATH=/usr/local/bin:/usr/bin:/bin
+SHELL=/usr/bin/bash
+$EXPORTER_CRON_SCHEDULE /app/exporter-scripts/scheduled-script.sh
+EOF
```

Comment:
This doesn't work for me. I get 

> waveform-exporter-1    | + cat
> waveform-exporter-1    | + crontab -
> waveform-exporter-1    | "-":2: bad command
> errors in crontab file, can't install.
> exec cron -f

Reply:
Is your value for `EXPORTER_CRON_SCHEDULE` set? The script checks for it being set but for some reason this check doesn't seem to work for me and gives the same error you get.